README.md

```diff
@@ -168,6 +168,63 @@ class CardSection extends React.Component {
 export default CardSection;
 ```
 
+### Using the `PaymentRequestButtonElement`
+
+The [Payment Request Button](https://stripe.com/docs/elements/payment-request-button) lets you collect payment and address information from your customers using Apple Pay and the Payment Request API.
+
+To use the `PaymentRequestButtonElement` you need to first create a [`PaymentRequest` Object](https://stripe.com/docs/stripe.js#the-payment-request-object). You can then conditionally render the `PaymentRequestButtonElement` based on the result of `paymentRequest.canMakePayment` and pass the `PaymentRequest` Object as a prop.
+
+```js
+class PaymentRequestForm extends React.Component {
+  constructor(props) {
+    super(props);
+
+    // For full documentation of the available paymentRequest options, see:
+    // https://stripe.com/docs/stripe.js#the-payment-request-object
+    const paymentRequest = props.stripe.paymentRequest({
+      country: 'US',
+      currency: 'usd',
+      total: {
+        label: 'Demo total',
+        amount: 1000,
+      },
+    });
+
+    paymentRequest.on('token', ({complete, ...data}) => {
+      console.log('received Stripe token': data);
```

Comment:
The token would be `data.token`, right? How about, `Received customer information:`

And maybe a separate log for the token?